README.md

```diff
@@ -1,60 +1,48 @@
-
-For example, suppose you are using the class `BetterHtml::Parser` exported from the `better_html` gem. Just doing a `require "better_html"` (which is the default require) does not load that type:
-
-```shell
-$ bundle exec pry
-[1] pry(main)> require 'better_html'
-=> true
-[2] pry(main)> BetterHtml
-=> BetterHtml
-[3] pry(main)> BetterHtml::Parser
-NameError: uninitialized constant BetterHtml::Parser
-from (pry):3:in `__pry__`
-[4] pry(main)> require 'better_html/parser'
-=> true
-[5] pry(main)> BetterHtml::Parser
-=> BetterHtml::Parser
-```
-
-In order to make sure that `tapioca` can reflect on that type, we need to add the line `require "better_html/parser"` to the `sorbet/tapioca/require.rb` file. This will make sure `BetterHtml::Parser` is loaded into memory and a type annotation is generated for it in the `better_html.rbi` file. If this extra `require` line is not added to `sorbet/tapioca/require.rb` file, then the definition for that type will be missing from the RBI file.
+# Tapioca - The swiss army knife of RBI generation
 
-If you ever run into a case, where you add a gem or update the version of a gem and run `tapioca gem` but don't have some types you expect in the generated gem RBI files, you will need to make sure you have added the necessary requires to the `sorbet/tapioca/require.rb` file.
-
-You can use the command `tapioca require` to auto-populate the `sorbet/tapioca/require.rb` file with all the requires found
-in your application. Once the file generated, you should review it, remove all unnecessary requires and commit it.
-
-## How does tapioca compare to "srb rbi gems" ?
+![Build Status](https://github.com/Shopify/tapioca/workflows/CI/badge.svg)
 
-[Please see the detailed answer on our wiki](https://github.com/Shopify/tapioca/wiki/How-does-tapioca-compare-to-%22srb-rbi-gems%22-%3F)
+Tapioca makes it easy to work with [Sorbet](https://sorbet.org) in your codebase. It surfaces types and methods from many sources that Sorbet cannot otherwise see – such as gems, Rails and other DSLs – compiles them into [RBI files](https://sorbet.org/docs/rbi) and makes it easy for you to add gradual typing to your application.
+
+**Features**:
+
+* Easy installation and configuration
+* Generation of RBI files for the gems used in your application
+  * Automatic generation from your application's Gemfile
+  * Importing of signatures from the gems source code
+  * Importing of documentation from the gems source code
```

Comment:
`gems source code` is not grammatically correct, since it is the "source code" that belongs to "gems", so should be possesive. Thus, it should either be `gems' source code` or `source code of gems`. I prefer the latter:
```suggestion
  * Importing of signatures from the source code of gems
  * Importing of documentation from the source code of gems 
```